### PR TITLE
Fix womtool endpoint to understand nonempty arrays

### PR DESCRIPTION
**Pull request by aednichols**

### CODE REVIEW

File: services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl [hunk 1]
```diff
@@ -7,6 +7,8 @@ workflow arrays {
     Array[Int]? optional_array
     Array[Int?] array_of_optionals
     Array[Int?]? optional_array_of_optionals
+    Array[Int]+ nonempty_array
+    Array[Int]+? optional_nonempty_array
```
Comment (salonishah11) [inline services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl]: What is `optional non-empty array`? Is it that if you have an array it should be non-empty, else no array at all? Similar to `Option[Array[Int]+]`?

Comment (aednichols) [reply services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl]: Yes, that's right - an array that may or may not be defined - but if it is, then it must be nonempty.

File: services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl [hunk 2]
```diff
@@ -17,6 +19,8 @@ workflow arrays {
     Array[Int]? output_optional_array = []
     Array[Int?] output_array_of_optionals = []
     Array[Int?]? output_optional_array_of_optionals = []
+    Array[Int]+ output_nonempty_array = [5]
```
Comment (salonishah11) [inline services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl]: Can you also explain `output non-empty array`?

File: services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl (hunk 1 above)
Comment (salonishah11) [reply services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl]: Sounds good. Thank you for the explanation!

File: services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl (hunk 2 above)
Comment (aednichols) [reply services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl]: I stuck an "output" on just to distinguish it from the input variables with the same names, which is helpful when looking at the JSON

Comment (salonishah11) [reply services/src/test/resources/describe/wdl_1_0/arrays/workflow.wdl]: Thank you for the explanation @aednichols! :)

**Pull request merged**